server/controllers/ArticleController.js

```diff
@@ -98,6 +99,7 @@ class ArticleController {
       const articleObj = article.dataValues;
       articleObj.tags = articleControllerHelper
         .formatTagResponse(article.tags);
+      articleObj.likesCount = articleObj.likes.length;
```

Comment:
I thought the database had a method for doing the count ? can you check, if it does let's use that instead

Reply:
Do you mean getting the count with sequelize? cos i tried doing something of this nature `attributes: [[sequelize.fn('COUNT', sequelize.col('ArticleLike.id')), 'likesCount']]` but was getting some errors